src/main/java/com/dominik/typer/service/matchresultpersistence/MatchResultService.java

```diff
@@ -38,4 +44,13 @@ private void checkIfMatchExists(Integer matchId) {
             throw new MyAppException("Match with id: " + matchId + " not found");
         }
     }
+    public MatchOutcome getMatchOutcome(MatchResult matchResult) {
+        if (matchResult.getTeam1Score() > matchResult.getTeam2Score()) {
```

Comment:
tak samo tutaj (do modelu metoda... , też zwracasz pewną klasę modelową 'enuma' względem innej wartości w modelu) więc można to też tam przenieść

Reply:
fixed